packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts

```diff
@@ -1012,3 +1017,8 @@ function assertNoLoaderParamsWithoutLoader(dir: NgOptimizedImage, imageLoader: I
             `To fix this, provide a custom loader or remove the \`loaderParams\` attribute from the image.`));
   }
 }
+
+
+function round2Decimals(numb: number): number {
+  return Math.round((numb + Number.EPSILON) * 100) / 100;
+}
```

Comment:
Quick question: why not keeping the `.toFixed(2)` as you had before?
If that was needed to keep integers untouched (avoid adding `.00`), we can do something like this:
```suggestion
function round(input: number): number {
  return Number.isInteger(input) ? input : input.toFixed(2);
}
```

Reply:
Yeah that was it, thanks !